src/styles/main.scss

```diff
@@ -1,3 +1,41 @@
+@import "./utils/variable.scss";
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./blocks/header.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Arial", "Helvetica", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  color: $color-general-text;
+}
+
 body {
   margin: 0;
 }
+
+h2 {
+  margin-top: 40px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.main {
+  padding: 50px 0 25px;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  width: getWidth($card-width, $card-right-shift,$cards-row);
```

Comment:
you've probably haven't added a file with functions here, that's why it is not working

Reply:
yes exactly) i forgot to do it